pkg/cmd/server/api/types.go

```diff
@@ -279,6 +279,21 @@ type MasterConfig struct {
 
 	// NetworkConfig to be passed to the compiled in network plugin
 	NetworkConfig MasterNetworkConfig
+
+	// JenkinsPipelineConfig holds information about the default Jenkins template
+	// used for JenkinsPipeline build strategy.
+	JenkinsPipelineConfig JenkinsPipelineConfig `json:"jenkinsPipelineConfig"`
```

Comment:
You are not annoying, I meant it good way. I appreciate the ping, otherwise
I will loose that comment (even my gmail magic sucks with this amount of
commits/comments).

On Tue, Apr 5, 2016 at 10:12 AM, Jimmi Dyson <notifications@github.com>
wrote:

> In pkg/cmd/server/api/types.go
> <https://github.com/openshift/origin/pull/7949#discussion_r58501919>:
>
> > @@ -279,6 +279,21 @@ type MasterConfig struct {
> >
> >  	// NetworkConfig to be passed to the compiled in network plugin
> >  	NetworkConfig MasterNetworkConfig
> > +
> > +	// JenkinsPipelineConfig holds information about the default Jenkins template
> > +	// used for JenkinsPipeline build strategy.
> > +	JenkinsPipelineConfig JenkinsPipelineConfig `json:"jenkinsPipelineConfig"`
>
> Don't mean to be annoying - there's a lot of noise to wade through each
> day & didn't want this to get missed.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7949/files/e2d6ffa012a680b0d57df505a1dd9ace775dcdf3#r58501919>
>



-- 
-----------------------------------------------
Michal Fojtik <mi@mifo.sk <mfojtik@mifo.sk>>
Red Hat OpenShift, Engineering